features/graph/documentation/README.md

```diff
@@ -1,38 +1,82 @@
+- [Access secrets with Netlify Graph](#access-secrets-with-netlify-graph)
+- [Netlify Graph and Next.js](#netlify-graph-and-nextjs)
 - [Feedback](#feedback)
+- [Learn more](#learn-more)
 
 ## Overview
 
-Netlify Graph is a new feature released through [Netlify Labs](https://www.netlify.com/blog/2021/03/31/test-drive-netlify-beta-features-with-netlify-labs/), that enables developers to seamlessly integrate third-party APIs in their web applications without having to write API-specific code.
+Netlify Graph enables developers to seamlessly integrate third-party APIs and services into their web applications without writing API-specific code. Instead of connecting different APIs and SDKs with brittle code, you can now abstract all that behind a convenient GraphQL interface.
+
+![Introduction to Netlify Graph.](../../../media/graph/graph-intro.png)
+
+Essentially, Netlify handles the messy integration work so you can focus on solving other problems.
+
+Netlify Graph and its component features are available for all sites in your Netlify team by default.
+
+## Requirements
+
+Netlify Graph uses the Netlify CLI to communicate between your local project and your Netlify site. Before you do anything else, you should install the latest version of the Netlify CLI.
+
+``` bash
+npm install netlify-cli -g
+```
 
-We handle the messy integration work, and you can focus on solving interesting problems. Instead of connecting different APIs and SDKs with brittle code, you can now abstract all that out behind a convenient GraphQL interface.
+## Example repo and tutorial
 
-## Components
+<!-- TODO: Update this section with link to the gravity repo -->
+If you'd like to try Netlify Graph in a test project, check out the `gravity` sample repo. It has a full tutorial that walks you through getting the `gravity` site running on Netlify to creating and using GraphQL queries.
 
-Netlify Graph in its current state has three core components, that you can learn more about by reading related documentation pages:
+## Get started with Netlify Graph
 
-- [API Management](api-management.md)
-- [API Explorer](api-explorer.md)
-- [API Authentication](api-authentication.md)
+  1. [Link your local project directory to your Netlify site](authentication.md#link-your-local-project-directory-to-your-netlify-site)
+  2. [Connect to your first API or service](authentication.md#connect-to-your-first-api-or-service) in the Netlify UI.
+  3. [Start a local CLI session with Netlify Dev](authentication.md#start-a-local-cli-session-with-netlify-dev).
+  4. [Create a GraphQL query with Graph Explorer](authentication.md#create-a-graphql-query-with-graph-explorer).
+  5. [Generate a query handler](authentication.md#generate-a-query-handler).
+  6. [Use a query handler in your project](authentication.md#use-a-query-handler-in-your-project).
 
-## Enabling the feature
+## Queries, mutations, and subscriptions
+<!-- QUESTION: Are we going to address fragments? -->
```

Comment:
@localden, do we want to address fragments in the docs at all?

Reply:
@KyleBlankRollins let's make this a fast-follow.